templates/zerver/help/message-info.md

```diff
@@ -1,4 +1,4 @@
-# Viewing information about a message
+# Viewing message information
```

Comment:
Since this change is unrelated you will probably have to move to another commit.